docs/source/Server_Ips/server_ips_1_16.rst

```diff
@@ -8,6 +8,17 @@ Server Ip's 1.16.4
 
 ----
 
+Vanilla 1.16.5 (McMMO)
+^^^^^^^^^^^^^^^^^
```

Comment:
Formatting needs fixing here

Reply:
Fixed. Sneaky little one that i missed, thanks!

Reply:
The ^ symbol needs to be the same length as the server name. Think you are missing 5 ^ arrows. Otherwise build fails.